docs/docs/using-graphql-playground.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Using the GraphQl Playground
+---
+
+## Intro
+
+In this page we want to introduce you to an alternative to the current IDE for your GraphQl queries: The [Prisma Playground](https://github.com/prisma/graphql-playground).
+
+## What is The Prisma Playground?
+
+The Playground is a way for you to interact with the data your sources and plugins add as schemas. You will be interacting with this data a lot and the Playground will help you greatly with exploring this data.
+
+## Accessing the Playground
+
+As this is an experimental feature, you need to access it by adding an environmental variable, you need to add `GATSBY_GRAPHQL_IDE` to your start script, like this:
+
+```
+"start": "GATSBY_GRAPHQL_IDE=playground npm run develop",
```

Comment:
maybe we should add this to `develop` script (as this script actually assume that `develop` is a script)?
```suggestion
"develop": "GATSBY_GRAPHQL_IDE=playground gatsby develop",
```
?

Reply:
But it's not a blocker I think, we can iterate on docs. But feature itself was tested and work so I'll go ahead and merge and publish

Reply:
Good catch, committed that